stdlib/Markdown/docs/src/index.md

```diff
@@ -277,14 +279,20 @@ elements.
 ### Display equations
 
 Large ``\LaTeX`` equations that do not fit inline within a paragraph may be written as display
-equations using a fenced code block with the "language" `math` as in the example below.
+equations using a fenced code block with the "language" `math`, or a double `$$`, as in the examples below.
 
 ````julia
 ```math
 f(a) = \frac{1}{2\pi}\int_{0}^{2\pi} (\alpha+R\cos(\theta))d\theta
 ```
 ````
 
+```julia
+md"$$H = - \sum p(x) \log p(x)$$"
```

Comment:
Ee can just do
```suggestion
raw"$$H = - \sum p(x) \log p(x)$$"
```
In order to stop interpreting the dollars aa interpolation.

Reply:
Yeah, demonstrating this with the `@md_str` seems confusing. This block should be demonstrating just Markdown, so it could just be

````
```markdown
$$ H = - \sum p(x) \log p(x) $$
```
````

maybe? Although, including this directly in a docstring would then fail because of the lack of escaping. But this is how you would have to write it if it's in a Markdown file.